mmdet3d/models/backbones/dgcnn.py

```diff
@@ -0,0 +1,94 @@
+from mmcv.runner import BaseModule, auto_fp16
+from torch import nn as nn
+
+from mmdet3d.ops import DGCNNFAModule, build_gf_module
+from mmdet.models import BACKBONES
+
+
+@BACKBONES.register_module()
+class DGCNN(BaseModule):
+    """Backbone network for DGCNN.
+
+    Args:
+        in_channels (int): Input channels of point cloud.
+        num_samples (tuple[int]): The number of samples for knn or ball query
+            in each GF module.
+        knn_mods (tuple[str]): If knn, mod of KNN of each GF module.
```

Comment:
Should be `knn_modes`? Please also replace all the `mod` with `mode` in the remaining code.

Reply:
should use knn_modes.